unyt/_unit_lookup_table.py

```diff
@@ -544,7 +553,29 @@
         ("hp", ("horsepower",)),
         ("oz", ("ounce",)),
         ("cal", ("calorie",)),
-        ("BTU", ("british_thermal_unit",)),
+        (
+            "BTU",
+            (
+                "british_thermal_unit",
+                "btu",
```

Comment:
Good catch on this alias, I didn't realize the automatic aliases we create to handle capitalization in unit names explicitly excludes all-upper-case names. It makes sense it works that way though, as well as adding the explicit aliases you're adding here.